src/Components/Breadcrumbs.js

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import { Link } from 'react-router-dom';
+import { AngleRightIcon } from '@patternfly/react-icons'
+
+export default function Breadcrumbs() {
+    return (
+        <nav className="pf-c-breadcrumb" aria-label="breadcrumb">
+            <ol className="pf-c-breadcrumb__list">
+                <li className="pf-c-breadcrumb__item">
+                    <Link to="/" className="pf-c-breadcrumb__link">Products</Link>
+                    <span className="pf-c-breadcrumb__item-divider">
+                        <AngleRightIcon />
+                    </span>
+                </li>
+                <li className="pf-c-breadcrumb__item">
+                    <span className="pf-c-breadcrumb__link pf-m-current" aria-current="page">
+                        Versions
+                    </span>
+                </li>
+            </ol>
+        </nav>
```

Comment:
In pf4 you can do something like:
```
import React from 'react';
import { Breadcrumb, BreadcrumbItem, BreadcrumbHeading } from '@patternfly/react-core';

SimpleBreadcrumbs = () => (
  <Breadcrumb>
    <BreadcrumbItem to="#">Section home</BreadcrumbItem>
    <BreadcrumbItem to="#">Section title</BreadcrumbItem>
    <BreadcrumbItem to="#">Section title</BreadcrumbItem>
    <BreadcrumbItem to="#" isActive>
      Section Landing
    </BreadcrumbItem>
  </Breadcrumb>
);
```

Reply:
@rabajaj0509 Yes, we can do this now.
When we started working on this app at that time the patternfly didn't implement the react-router with the breadcrumb and used product anchor tag with it and that was the reason we had to hardcode the breadcrumb component.